e3sm_to_cmip/util.py

```diff
@@ -54,6 +60,61 @@ def print_debug(e):
     traceback.print_tb(tb)
     print(e)
 
+def setup_logging(loglevel, logpath):
+    logname = logpath + "-" + UTC.localize(datetime.utcnow()).strftime("%Y%m%d_%H%M%S_%f")
+    if loglevel == "debug":
+        level = logging.DEBUG
+    elif loglevel == "error":
+        level = logging.ERROR
+    elif loglevel == "warning":
+        level = logging.WARNING
+    else:
+        level = logging.INFO
+    logging.basicConfig(
+        filename=logname,
+        # format="%(asctime)s:%(levelname)s:%(module)s:%(message)s",
+        format="%(asctime)s_%(msecs)03d:%(levelname)s:%(message)s",
+        datefmt="%Y%m%d_%H%M%S",
+        level=level,
+    )
+    logging.Formatter.converter = time.gmtime
+    # should be a separate message call
+    # logging.info(f"Starting up the warehouse with parameters: \n{pformat(self.__dict__)}")
```

Comment:
```suggestion
```

Usually, commented out code should not be committed unless there is good reason to include  it.

Reply:
Feel free to remove it.  I tend to overlook that stuff - sorry.